source/visionEnhancementProviders/NVDAHighlighter.py

```diff
@@ -246,7 +246,7 @@ class NVDAHighlighterGuiPanel(
 	_enableCheckSizer: wx.BoxSizer
 	_enabledCheckbox: wx.CheckBox
 	
-	helpId = "VisionSettingsFocusHighlight"
+	helpId = "VisionSettingsVisualHighlight"

```

Comment:
Don't change the help ID's these aren't really user visible, and are included in links. Changing them will break links.

Reply:
No, the links don't break because I've changed it everywhere. I can still get them back. But I have tested it and there is no breakage.

Reply:
Yes, IMO, you need to change them back in order to avoid translators to translate it again.

Reply:
External links will still break, links from outside of NVDA (EG from an GitHub issue, or someones blog). These anchor ID's aren't user visible, and could be anything, once set they should remain as stable as possible.